.gitignore

```diff
@@ -3,6 +3,7 @@ dist
 node_modules
 .nyc_output
 packages/adblocker-benchmarks/requests.json
+packages/adblocker-benchmarks/blockers
```

Comment:
Why `.gitignore` this folder? That's where all the entrypoints are for each adblocker being benchmarked.

Reply:
Sorry, I've updated `.gitignore` now. We only need to ignore the directories we are creating via scripts.